src/swell/suites/localensembleda/flow.cylc

```
@@ -213,7 +211,8 @@
 #        {%- endfor %}
 
     [[SaveObsDiags-{{model_component}}]]
-        script = "swell task SaveObsDiags $config -d $datetime -m {{model_component}}"
+        script = true
+#        script = "swell task SaveObsDiags $config -d $datetime -m {{model_component}}"
```

Am I understanding correctly that this effectively disables the `SaveObsDiags` module, because the module "runs" but doesn't actually do anything?

If so, this feels brittle and likely to cause confusion. If we're going to skip this task, we should properly remove it.

At an absolute minimum, instead of using the uninformative value `true`, we should do something like this (replacing `###` with a Swell GitHub issue to re-enable this). Since the `echo` command should silently succeed, this accomplishes the same result but in a more informative way.

```
script = "echo 'SaveObsDiags is disabled for this task. See GH issue ###'"
```

---

I agree, if you want to skip this task for now you can comment out the script  in [runtime] and [scheduling].

This task for other suites otherwise.

---

I think this is resolved now?

---

Right, that is now resolved.